h2o-docs/src/booklets/v2_2015/source/GLMBooklet.tex

```diff
@@ -571,6 +571,100 @@ \subsubsection{Tweedie Models}
 \centering
 \includegraphics[scale=0.5]{../../../product/images/tweedie_deviance.png}
 \end{figure}
+
+
+\paragraph{Tweedie Dispersion Parameter Estimation}
```

Comment:
```suggestion
\paragraph{Tweedie Parameters Estimation}
```

This section describes likelihood used for estimating Tweedie variance power(`p`) and dispersion(`phi`).